R/4.1.0_SoupX/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+LABEL maintainer="Nathan Johnson <johnsonn13@chop.edu>"
+LABEL description="Install R in Miniconda with SoupX package for background removal in single-cell RNA"
+
+ENV MINICONDA_VERSION=py38_4.8.3
+
+RUN apt update -y && apt install -y curl \
+    && curl -sL https://repo.anaconda.com/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh -o /tmp/miniconda.sh \
+    && sh /tmp/miniconda.sh -bfp /usr \
+    && conda config --add channels conda-forge \
+    && conda install -y -c conda-forge r-base=4.1.0 \
+    && R -e "install.packages(c('optparse', 'codetools', 'survival', 'plyr', 'zoo', 'data.table', 'htmlwidgets', 'lazyeval', 'reshape2', 'SoupX'), dependencies=TRUE, repos='http://cran.us.r-project.org')"
+
+ADD Dockerfile .
+
+CMD ["/bin/bash"]
```

Comment:
I am assuming this was added because when running as an interactive Docker, this will drop into an R shell instead of bash shell, right? If so, this might actually end up creating problems with cavatica - I think it ends up doing something like `/bin/sh /bin/bash` then whatever cwl command you have , which ends up failing completely. I'd leave this out